.ci/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I suggest to rename directory `.ci/` to `test/ci/`, so it doesn't have to be hidden, and doesn't clutter the GitHub repo.